### DISCUSSION TITLE
Enhancement of existing FUOTA code to support latest version of specifications

### DISCUSSION THREAD

**Discussion by praneet-kaur** [Lora-net/LoRaMac-node#1244] [Q&A] [answered]
I was wondering if there is a plan to upgrade the FUOTA packages to latest version of the specifications.
The version 2 of clock synchronisation, fragmentation and remote multicast package specifications is available through LoRa Alliance website.

**Comment by mluis1** [comment, marked as answer]
Those documents have not been publicly released by the LoRa-Alliance.

You shouldn't have uploaded those documents here.

Once the LoRa-Alliance publicly releases those documents we intend to update this repository. However we can not provide a time frame on when the update will be available.

Please, if possible remove those documents as well as the internal LoRa Alliance website folder link.

**Reply by mluis1** [reply]
I have edited the original post and removed the files links from the post history.

**Reply by praneet-kaur** [reply]
Thanks @mluis1! Apologies for the inconvenience.

**Reply by mluis1** [reply]
No problem and thank you for your understanding.
